### ISSUE TITLE
Cannot download data from SPB exchange using scripts

### ISSUE THREAD

**Issue by kvirolainen** [ffeast/finam-export#7]
E.g. ticker SIG.

```
>>./scripts/finam-lookup.py --contract=SIG
INFO:finam.export:Fetching https://www.finam.ru/profile/moex-akcii/gazprom/export/
INFO:finam.export:Fetching https://www.finam.ru/cache/N72Hgd54/icharts/icharts.js
DEBUG:finam.export:Parsing line starting with "var aEmitentIds = [4"
DEBUG:finam.export:Parsing line starting with "var aEmitentNames = "
DEBUG:finam.export:Parsing line starting with "var aEmitentCodes = "
DEBUG:finam.export:Parsing line starting with "var aEmitentMarkets "
                           name code  market
id                                          
874943  Signet Jewelers Limited  SIG      25
481495  Signet Jewelers Limited  SIG     517
```

```
>>./scripts/finam-download.py --market=SPB --destdir=. --contracts=SIG --timeframe=TICKS --startdate=2020-03-27 --enddate=2020-03-27
INFO:root:Handling SIG
INFO:finam.export:Fetching https://www.finam.ru/profile/moex-akcii/gazprom/export/
INFO:finam.export:Fetching https://www.finam.ru/cache/N72Hgd54/icharts/icharts.js
DEBUG:finam.export:Parsing line starting with "var aEmitentIds = [4"
DEBUG:finam.export:Parsing line starting with "var aEmitentNames = "
DEBUG:finam.export:Parsing line starting with "var aEmitentCodes = "
DEBUG:finam.export:Parsing line starting with "var aEmitentMarkets "
INFO:__main__:Downloading contract id                         874943
name      Signet Jewelers Limited
code                          SIG
market                         25
Name: 0, dtype: object
INFO:finam.export:Fetching http://export.finam.ru/table.csv?sep=3&at=1&e=.csv&d=d&f=table&dtf=1&MSOR=0&tmf=3&mstimever=1&mstime=on&sep2=1&em=874943&code=SIG&cn=SIG&df=27&yf=2020&dt=27&datf=6&yt=2020&market=25&mf=2&mt=2&p=1
ERROR:__main__:Returned data doesnt seem like a valid csv dataset: 
```

The download URL is for market=25, not for market=517 (SPB)

**Comment by ifeelgood** [comment]
@ffeast , can you please approve the pull request mentioned above?
I would like to remove my fork where I fixed it.

**Comment by ffeast** [comment]
@ifeelgood  merged, released 2.0.0 https://pypi.org/project/finam-export/
@kvirolainen your case seems to work now (see the output below)
```
finam-download.py --skiperr=0 --startdate=2001-01-01 --market=SPB --contracts=SIG  --destdir=./tmp
INFO:root:Handling SIG
INFO:finam.export:Fetching https://www.finam.ru/profile/moex-akcii/gazprom/export/
INFO:finam.export:Fetching https://www.finam.ru/cache/N72Hgd54/icharts/icharts.js
DEBUG:finam.export:Parsing line starting with "var aEmitentIds = [4"
DEBUG:finam.export:Parsing line starting with "var aEmitentNames = "
DEBUG:finam.export:Parsing line starting with "var aEmitentCodes = "
DEBUG:finam.export:Parsing line starting with "var aEmitentMarkets "
INFO:__main__:Downloading contract id                         481495
name      Signet Jewelers Limited
code                          SIG
market                        517
Name: 0, dtype: object
INFO:finam.export:Fetching http://export.finam.ru/table.csv?sep=3&at=1&e=.csv&d=d&f=table&dtf=1&MSOR=0&tmf=3&mstimever=1&mstime=on&sep2=1&em=481495&code=SIG&cn=SIG&df=1&yf=2001&dt=10&datf=5&yt=2020&market=517&mf=0&mt=4&p=8
INFO:__main__:Sleeping for 1 second(s)
(test1) oleg@MacBook-Air-Oleg trading % hed a
(test1) oleg@MacBook-Air-Oleg trading % head ./tmp/SIG-DAILY.csv 
index,<OPEN>,<HIGH>,<LOW>,<CLOSE>,<VOL>
2017-06-14,59.38,60.13,58.38,60.0,517390
2017-06-15,60.18,60.43,59.07,59.91,420176
2017-06-16,59.72,60.45,58.31,60.45,387100
2017-06-19,61.1,61.1,60.04,60.85,411138
2017-06-20,60.99,60.99,59.44,59.5,415444
2017-06-21,59.5,59.5,55.64,56.2,399232
2017-06-22,56.56,57.4,55.61,56.98,210267
2017-06-23,56.69,58.48,55.87,58.42,286387
2017-06-26,58.25,60.47,58.18,60.26,235678

```
